### PR TITLE
chore(abci): update comments that do not start with the name of the exported element

### PR DESCRIPTION
**Pull request by hoanguyenkh**
Refactor code and update comments

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments
- [x] Title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

### CODE REVIEW

File: abci/types/util.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func (v ValidatorUpdates) Len() int {
 	return len(v)
 }
 
-// XXX: doesn't distinguish same validator with different power.
+// Less doesn't distinguish the same validator with different power.
```
Comment (cason) [inline abci/types/util.go:26]: I wouldn't remove the tag here. This should be a FIXME, TODO?

**Review by cason — COMMENTED** [review]
Thank you for that, left some comments.

File: abci/types/util.go (hunk 1 above)
Comment (hoanguyenkh) [reply abci/types/util.go]: i don't know what reason is, let me rollback the comment

**Review by melekes — APPROVED** [review]
👍

**Comment by melekes** [comment]
Thanks @hoanguyenkh ❤️

**Pull request merged**